src/state/slices/marketDataSlice/selectors.ts

```diff
@@ -15,16 +15,17 @@ import { selectAssetIdParamFromFilter } from '@/state/selectors'
 import type { PriceHistoryData } from '@/state/slices/marketDataSlice/types'
 import { selectSelectedCurrency } from '@/state/slices/preferencesSlice/selectors'
 
-export const selectMarketDataIdsSortedByMarketCapUsd = (state: ReduxState) =>
```

Comment:
Actually didn't need to be exported, and none of the others root selectors to - used destructuring here instead to keep things tidy